perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -405,10 +436,10 @@ def WriteObjectFromBuffer(storage_schema, bucket_name, object_name,
     if host_to_connect is not None:
```

Comment:
I added the seek, but moved it to before the timing starts, because I don't think it should be part of the operation latency. Other than that it's exactly like you suggested.